src/location.mli

```diff
@@ -42,6 +42,8 @@ type nonrec 'a loc = 'a loc =
   ; loc : t
   }
 
+val mknoloc : 'a -> 'a loc
```

Comment:
Could you add documentation? I think it would be good also to stress in the documentation that this returns a ghost located value and should be avoided and only be used if one really doesn't need to locate the value.